src/containers/Sidebar/SidebarBottomItem.js

```diff
@@ -31,7 +30,7 @@ const SidebarBottomItem = ({ item, sidebarLogo, toggleSidebarModal }) => {
 			}
 		}
 	}
-
+SidebarBottomItem
```

Comment:
@petrKavulok wtf is this? Did you tested the functionality? if so, the how come your app did not crash?

Reply:
@Pe5h4 I guess i copy pasted the components name by mistake here. To my complete surprise, it works just fine :D